core/src/main/java/org/opensearch/sql/storage/read/TableScanBuilder.java

```diff
@@ -108,4 +108,7 @@ public boolean pushDownHighlight(LogicalHighlight highlight) {
   public <R, C> R accept(LogicalPlanNodeVisitor<R, C> visitor, C context) {
     return visitor.visitTableScanBuilder(this, context);
   }
+
+  public void pushDownOffset(int i) {
```

Comment:
no return value means always push down successfully?

Reply:
This is reverted, not pushdown anymore